src/Modules/Station/Components/AddSong/PreviewVideo/PreviewVideo.scss

```diff
@@ -31,6 +31,14 @@
     margin-left: 5px;
   }
 
+  .warning {
+    color: red;
+  }
+
+  &__tooltipModify {
```

Comment:
You shouldn't use this modify, it will cause some UI Break when it responsive.
Instead, you can change placement of tooltip from `bottom` to `right`, it will be better.
Try it!!!